portfolio/src/main/webapp/script.js

```diff
@@ -45,26 +45,60 @@ function openConnectModal() {
   modal.style.display = "block";
 }
 
-function closeIntroModal(){
+function openQuestionModal() {
+  const modal = document.getElementById("myQuestionModal");
+  modal.style.display = "block";
+}
+
+function openViewModal() {
+  const modal = document.getElementById("myQuestionViewModal");
+  modal.style.display = "block";
+  fetch('/data').then(response => response.json()).then((Questions) => {
+  const questionContainer = document.getElementById("question-container");
+  questionContainer.innerHTML = '';
```

Comment:
let's use `""` instead of `''` to make your quotes consistent. You've been using double quotes ("") everywhere else.  try not to mix usages of single and double quotes. whatever you choose it needs to be consistent